pkg/yurttunnel/trafficforward/iptables/iptables.go

```diff
@@ -534,64 +535,17 @@ func (im *iptablesManager) syncIptableSetting() {
 }
 
 func (im *iptablesManager) getAddedAndDeletedNodes(currentNodesIP []string) (bool, []string, []string) {
-	changed := false
-	if len(im.lastNodesIP) != len(currentNodesIP) {
-		changed = true
-	}
-	addedNodesIP := make([]string, 0)
-	for i := range currentNodesIP {
-		found := false
-		for j := range im.lastNodesIP {
-			if currentNodesIP[i] == im.lastNodesIP[j] {
-				found = true
-				break
-			}
-		}
-
-		if !found {
-			addedNodesIP = append(addedNodesIP, currentNodesIP[i])
-			changed = true
-		}
-	}
-
-	deletedNodesIP := make([]string, 0)
-	for i := range im.lastNodesIP {
-		found := false
-		for j := range currentNodesIP {
-			if im.lastNodesIP[i] == currentNodesIP[j] {
-				found = true
-				break
-			}
-		}
+	currentNodesIPSet := sets.NewString(currentNodesIP...)
+	lasNodesIpSet := sets.NewString(im.lastNodesIP...)
 
-		if !found {
-			deletedNodesIP = append(deletedNodesIP, im.lastNodesIP[i])
-			changed = true
-		}
-	}
+	changed := currentNodesIPSet.Equal(lasNodesIpSet) && lasNodesIpSet.Equal(currentNodesIPSet)
```

Comment:
Isn't it enough to call Equal once? Like `changed := currentNodesIPSet.Equal(lasNodesIpSet)`